sdk/openai/Azure.AI.OpenAI/tests/ChatCompletionsTests.cs

```diff
@@ -81,6 +81,65 @@ public async Task ChatCompletionsContentFilterCategories(Service serviceTarget)
             AssertExpectedContentFilterResponseResults(firstChoice.ContentFilterResults, serviceTarget);
         }
 
+        [RecordedTest]
+        [TestCase(Service.Azure)]
+        [TestCase(Service.NonAzure)]
+        public async Task ChatCompletionsLogprobs(Service serviceTarget)
```

Comment:
I'd suggest to combine these two new added tests into one since they have a lot overlap. We can keep `ChatCompletionsTopLogprobs` and rename to `ChatCompletionsLogprobs`.

Reply:
Resolved by https://github.com/ninghu/azure-sdk-for-net/pull/15/commits/3f429c372203e07a2d2556059ee39edffb2b55e4